src/software/proto/BUILD

```diff
@@ -17,6 +17,28 @@ proto_library(
     visibility = ["//visibility:private"],
 )
 
+proto_library(
```

Comment:
perhaps put a link with the commit as a comment in case we need to go back and update it

Reply:
Done